ogr/services/github/service.py

```diff
@@ -77,12 +77,17 @@ def github_app_private_key(self):
         return None
 
     def __str__(self) -> str:
-        token_str = f", token='{self.token}'" if self.token else ""
+        token_str = (
+            f", token='{self.token[:1]}***{self.token[-1:]}'" if self.token else ""
```

Comment:
those `:1` and `-1:` are clever, but do we need them (instead of more simple `0` and `-1`) when we have the `if self.token` ?

Reply:
You are right. Do you want me to "fix! that in another PR?